components/FilterForm/index.js

```diff
@@ -50,43 +49,36 @@ export default function FilterForm({ onFilterUpdate, plants }) {
         onChange={(e) => setPlantSize(e.target.value)}
       >
         <option value="">Select Size</option>
-        <option value="Small to Medium">Small to Medium</option>
-        <option value="Medium">Medium</option>
-        <option value="Medium to Large">Medium to Large</option>
-        <option value="Large">Large</option>
+        <option value="Small">Small (15cm-50cm)</option>
+        <option value="Medium">Medium (50cm-90cm)</option>
+        <option value="Large">Large (90cm+)</option>
       </StyledSelect>
       <StyledSelect
         name="sunlightRequirement"
         onChange={(e) => setSunlightRequirement(e.target.value)}
       >
         <option value="">Select Sunlight Requirement</option>
-        <option value="Low to Medium Light">Low to Medium Light</option>
-        <option value="Low to Bright Light">Low to Bright Light</option>
-        <option value="Bright Indirect Light">Bright Indirect Light</option>
-        <option value="Bright Direct Light">Bright Direct Light</option>
         <option value="Full Sun">Full Sun</option>
+        <option value="Indirect Light">Indirect Light</option>
+        <option value="Half-Shade">Half-Shade</option>
       </StyledSelect>
       <StyledSelect
         name="waterNeeds"
         onChange={(e) => setWaterNeeds(e.target.value)}
       >
         <option value="">Select Water Needs</option>
-        <option value="Low">Low</option>
-        <option value="Low to Medium">Low to Medium</option>
-        <option value="Medium">Medium</option>
-        <option value="Medium to High">Medium to High</option>
-        <option value="High">High</option>
+        <option value="Weekly">Weekly</option>
+        <option value="Every other week">Every other week</option>
+        <option value="Monthly">Monthly</option>
       </StyledSelect>
       <StyledSelect
         name="optimalTemperature"
         onChange={(e) => setOptimalTemperature(e.target.value)}
       >
         <option value="">Select Temperature</option>
-        <option value="15-27°C">15-27°C</option>
+        <option value="15-20°C">15-20°C</option>
         <option value="18-25°C">18-25°C</option>
-        <option value="18-24°C">18-24°C</option>
         <option value="20-30°C">20-30°C</option>
-        <option value="15-30°C">15-30°C</option>
```

Comment:
Instead of using numbers you could also use as values "low", "average" and "high" as I think this easier to maintain. For example: If you want to change the degrees on some point this might not need in changing the `value` of each `option` too.